extensions/wikia/AdEngine/js/template/bfaa.js

```diff
@@ -78,6 +80,12 @@ define('ext.wikia.adEngine.template.bfaa', [
 				page = doc.getElementsByClassName('WikiaSiteWrapper')[0];
 				wrapper = doc.getElementById('WikiaTopAds');
 				handler = desktopHandler;
+
+				if (params.uap) {
+					uapContext.setUapId(params.uap);
+					bottomLeaderboard.init();
```

Comment:
Moved (If we 100% sure that `initDesktopSlots` will be executed after `BFAA` will set uap id).